app/assets/stylesheets/application.scss

```diff
@@ -10,5 +10,4 @@
 
 // Your CSS partials
 @import "components/index";
-@import "pages/index";
 @import "pages/home";
```

Comment:
keep index, delete home ?

Reply:
because in home there's only one line "@import index"..., if we need to do sth with that later i will add :|